app/internal/page/how_should_replacement_attorneys_step_in.go

```diff
@@ -51,7 +52,7 @@ func HowShouldReplacementAttorneysStepIn(tmpl template.Template, lpaStore LpaSto
 
 				redirectUrl := taskListPath
 
-				if len(lpa.Attorneys) > 1 && lpa.HowAttorneysMakeDecisions == "jointly-and-severally" && lpa.HowShouldReplacementAttorneysStepIn == "none" {
+				if len(lpa.Attorneys) > 1 && lpa.HowAttorneysMakeDecisions == JointlyAndSeverally && lpa.HowShouldReplacementAttorneysStepIn == AllCanNoLongerAct {
```

Comment:
I think this should be

```
len(lpa.Attorneys) > 1 && 
lpa.HowAttorneysMakeDecisions == JointlyAndSeverally && 
lpa.HowShouldReplacementAttorneysStepIn == AllCanNoLongerAct && 
len(lpa.ReplacementAttorneys) > 1
```

then it matches the logic in complete

Reply:
Yeah, will update.

I was trying to come up with some sort of name to give the various checks some context but they all ended up being ridiculously long...feels like it would be nicer to wrap this up so we don't need to maintain it in two places. Will have a play around with some options.

Reply:
Scratch that - I forgot I'd scoped by attorney length. I'll drop in the logic.